### PR TITLE
Maintainer change + fix warnings

### LINKED ISSUE #318: Warning message: Could not find Sum-of-Squares for the (Intercept) in the ANOVA table.
Following easystats/easystats#338, I would like to get rid of all warnings in tests for `report`. I need help from someone knowledgeable on the matter to help clarify whether the warning below is expected (therefore to be suppressed with `verbose = FALSE`) or unexpected (therefore to be fixed). This is the first example in our tests for `report.aov.R`.

``` r
library(report)
model <- anova(lm(Sepal.Width ~ Species, data = iris))
r1 <- report(model)
#> Warning: Could not find Sum-of-Squares for the (Intercept) in the ANOVA table.
```

<sup>Created on 2023-01-16 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

### PR DESCRIPTION
**Pull request by rempsyc**
In this PR, I change the maintainer from @DominiqueMakowski to @rempsyc and fix or silence warnings in R CMD check.

---

Also closes #318

### CODE REVIEW

**Review by DominiqueMakowski — APPROVED** [review]
2023 is gonna be the year of report

**Comment by rempsyc** [comment]
> Merging #321 (34610eb) into main (245398b) will decrease coverage by 59.08%.

(not sure why Codecov says the code coverage suddenly drops to 0 as I just set `verbose = TRUE` at some places)

**Comment by rempsyc** [comment]
I wonder if we should turn off lintr for indentations because they often conflict with the line length requirement when you try to reduce the number of characters by jumping to the next line after the parenthesis...

File: DESCRIPTION [hunk 1]
```diff
@@ -38,7 +38,7 @@ Authors@R:
              role = "ctb",
              email = "mutlusun@users.noreply.github.com",
              comment = c(ORCID = "0000-0002-6021-804X")))
-Maintainer: Dominique Makowski <dom.makowski@gmail.com>
+Maintainer: Rémi Thériault <remi.theriault@mail.mcgill.ca>
```
Comment (IndrajeetPatil) [inline DESCRIPTION:41]: In addition to this, you will also need to change your role from `"aut"` to `c("aut", "cre")`.

Comment (IndrajeetPatil) [reply DESCRIPTION:41]: And remove `"cre"` from Dom's role. Maintainers are also expected to be the creators.

**Review by IndrajeetPatil — APPROVED** [review]
This is awesome work, Remi! Thanks a lot.

As for the lintr, the indetation issue should be fixed soon. You can ignore the for now. Also, note that lintr workflow doesn't fail if lints are found, so it shouldn't bother anyone.

File: DESCRIPTION (hunk 1 above)
Comment (rempsyc) [reply DESCRIPTION:41]: Weird, they should make another role like `mtn` or something 😤

**Comment by rempsyc** [comment]
Note: the `lint-changed-files ` workflow will still fail because I haven't addressed the  `Don't use nested ifelse()` requirement in `report_effectsize.R` since we decided earlier to postpone this until we have a working lookup table solution.

**Comment by rempsyc** [comment]
For `fatal: invalid reference: maintainer` in the Style workflow, I imagine this is because there is a mismatch on CRAN?

**Comment by IndrajeetPatil** [comment]
Style workflow is failing because you are making a PR from your fork instead of a new branch in the current repo.

I need to fix this: https://github.com/easystats/workflows/issues/26

**Pull request merged**